django/db/backends/ddl_references.py

```diff
@@ -186,7 +186,7 @@ def rename_column_references(self, table, old_column, new_column):
     def __str__(self):
         suffix = self.suffix_template % {
             "to_table": self.to_reference.table,
-            "to_column": self.to_reference.columns[0],
+            "to_column": "_".join(self.to_reference.columns),
```

Comment:
I think it's worth double-checking that we don't run into any weird length restrictions with this. I'm hoping that this is already handled by `create_fk_name`, but I haven't checked myself.

Reply:
Yeah the suffix is truncated later so it should be fine